.github/workflows/dockerhub.yml

```diff
@@ -0,0 +1,17 @@
+name: Docker Hub build
+on:
+  push:
+    branches:
+      - develop-foxy
+      - master
+      - "release/*"
+    tags:
+      - "carma-system-*"
+jobs:
+  dockerhub:
+    uses: usdot-fhwa-stol/actions/.github/workflows/dockerhub.yml@main

```

Comment:
This image needs to be tagged as develop-foxy

Reply:
Yes, This workflow will push the tag as develop-foxy as it take base branch name as tag